source/assets/public/grid/index.css

```diff
@@ -75,3 +62,23 @@
 .c-sponsor-level--size-5 .col-1 {
   width: 20%;
 }
+
+@media (max-width:639px) {
+  .grid {
+    margin-right: -2rem;
+  }
+  /* on phones stack all columns */
+  [class*='col-'],
+  .c-sponsor-level [class*='col-'] {
+    width: 100%;
+  }
+  [class*='col-'] {
+    padding-right: 2rem;
+  }
+}
+
+@media (min-width:640px) and (max-width:1023px) {
+  .c-sponsor-level--size-1 .col-1 {
+    width: 100%;
+  }
+}
```

Comment:
There's 🚫⏎ on a bunch of these files

Reply:
Could Github or Git be swallowing those newlines? I've got no idea why that's happening :|